ui/src/views/Quickstart/index.jsx

```diff
@@ -103,6 +158,22 @@ const cmdDirectory = (type, org = '<YOUR_ORG>', repo = '<YOUR_REPO>') =>
       ].join(' && '),
     ],
   }[type]);
+const initialState = {
```

Comment:
rather than having a global variable with component's state, it's a much better idea to save state in the component's class

Reply:
I tried moving the initialState inside class as static variable, but on accessing it in constructor using `QuickStart.initialState` it returns `undefined`.
I there something I need to do while defining static variables in class?

> When I press it, the `${match}` condition becomes empty, although pull-request-related flags are set.

I have fixed this part, the method I am using for this fix, might needs a review.
Also, this https://tools.taskcluster.net/quickstart does not supports resetting editor values which are changed manually.
In this I am also resetting value of editor to default if its getting change. Due to which the implementation is a bit different from the old one

Reply:
Usually class props are accessed using `this` keyword: `this.initialState`

It doesn't have to be static variable (in fact, I'm not sure if the transpiler we use here even allows for keyword `static`), but I would like to see it defined inside the class

> Also, this https://tools.taskcluster.net/quickstart does not supports resetting editor values which are changed manually.

You can add that functionality on top of the existing code, I believe

Reply:
> Usually class props are accessed using `this` keyword: `this.initialState`

I will work on this change.
Resetting I implemented.

Reply:
> It doesn't have to be static variable (in fact, I'm not sure if the transpiler we use here even allows for keyword `static`), but I would like to see it defined inside the class

I was using it as static, as it was similarly getting used in https://github.com/taskcluster/taskcluster-tools/blob/master/src/views/QuickStart/index.jsx#L94